purchases/src/test/java/com/revenuecat/purchases/PurchasesFactoryTest.kt

```diff
@@ -84,7 +84,12 @@ class PurchasesFactoryTest {
         verify(exactly = 1) { apiKeyValidatorMock.validateAndLog("fakeApiKey", Store.PLAY_STORE) }
     }
 
-    private fun createConfiguration(apiKey: String = "fakeApiKey"): PurchasesConfiguration {
-        return PurchasesConfiguration.Builder(contextMock, apiKey).build()
+    private fun createConfiguration(testApiKey: String = "fakeApiKey"): PurchasesConfiguration {
+        return mockk<PurchasesConfiguration>().apply {
+            every { context } returns contextMock
+            every { apiKey } returns testApiKey
+            every { appUserID } returns "appUserID"
+            every { store } returns Store.PLAY_STORE
+        }
```

Comment:
I could have split these tests... But I decided to just use a mock to not have to do that.